persons/index.html

```diff
@@ -1,11 +1,72 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
-    <div class="top-padding">
-        <div class="container">
-            <h1>Persons Module</h1>
-            <hr/>
-            <p>Use these pages to model your own subcollection pages.</p>
-            <h2>Example entries</h2>
-        </div>
+    <!--
```

Comment:
It looks like there's some d3 that should be kept in? But I'm not sure if the CSS classes are up to date.

Reply:
This is fine. We will need to update the data that is used by this graph at some point.